.idea/codeStyles/Project.xml

```diff
@@ -1,7 +1,12 @@
 <component name="ProjectCodeStyleConfiguration">
   <code_scheme name="Project" version="173">
     <JSCodeStyleSettings version="0">
```

Comment:
hummm I think we should configure our eslint to try matching these styles rules to keep the code pattern between IntelliJ and VSCode devs

Reply:
I thought everyone was using IntelliJ 😅 
I can have a look and configure eslint as well, but I would keep IntelliJ configurations as those are better integrated with the IDE and it is quicker to apply a fix if you fail following it.
For some of the configuration, we could also use [editorconfig](https://editorconfig.org) (which I already introduced), but it requires a plugin to work with VSCode